code/modules/shuttle/mini_dropship.dm

```diff
@@ -173,6 +173,10 @@
 	open_prompt = FALSE
 	clean_ui_user()
 
+/obj/machinery/computer/camera_advanced/shuttle_docker/minidropship/attack_hand(mob/user)
+	if(damaged == TRUE)
+		to_chat(user, "The computer blink and let some weirds sounds, its broken!")
```

Comment:
needs a span_warning(

Reply:
and a return

Reply:
ok going to do it

Reply:
can you say if it worked? i tryed to push it but this is my first time doing it